components/menu/apiExamples/loading.js

```diff
@@ -0,0 +1,5 @@
+export function auroMenuLoadingExample() {
+  document.querySelector("#loadingExampleToggleButton").addEventListener("click", () => {
+    document.querySelectorAll("#loadingExampleTable auro-menu").forEach(menu => menu.toggleAttribute("loading"));
```

Comment:
Should `aria-busy="true"` also be added?

```
menu.setAttribute("aria-busy", menu.hasAttribute("loading"));
```

Reply:
Just realized I wrote this on the example page, but I meant `auro-menu.js`

Reply:
added. thanks! :)